scripts/generate_pinval/generate_pinval.py

```diff
@@ -476,19 +476,17 @@ def _fmt_generic_numeric(x):
             return txt
         return x
 
-    # Recode character columns to human-readable values
-    if chars_recode:
-        chars_to_recode = [
-            col for col in df.columns if col.startswith("char_") and col != "char_apts"
-        ]
+    chars_to_recode = [
+        col for col in df.columns if col.startswith("char_") and col != "char_apts"
+    ]
 
-        df = ccao.vars_recode(
-            data=df.copy(),
-            cols=chars_to_recode,
-            code_type="long",
-            as_factor=False,
-            dictionary=ccao.vars_dict,
-        )
+    df = ccao.vars_recode(
+        data=df.copy(),
+        cols=chars_to_recode,
+        code_type="long",
+        as_factor=False,
+        dictionary=ccao.vars_dict,
+    )
```

Comment:
I deleted the conditional when I re-added this, I don't really see a scenario where we wouldn't want `as_factor=False,`